yarn-error.log

```diff
@@ -0,0 +1,11859 @@
+Arguments: 
```

Comment:
I think we can add this file to .gitignore.

Reply:
Agree, done, thank you.